tests/test_progress_report.py

```diff
@@ -0,0 +1,34 @@
+from oda_api.api import ProgressReporter
+import os
+import json
+
+callback_file = ".oda_api_callback"
+
+def test_progress_reporter_disabled():
+    if os.path.isfile(callback_file):
+        os.remove(callback_file)
+    # if callback is not available
+    pr = ProgressReporter()
+    assert not pr.enabled
+    pr.report_progress(stage='simulation', progress=50, substage='spectra', subprogress=30, message='some message')
```

Comment:
perhaps it should be verified that basically nothing happens?

Reply:
I am not sure what precisely you suggest to verify. In the test_progress_reporter_disabled() we check that the progress reporter doesn't fail if callback URL is not available. In this case the report_progress does nothing.

Reply:
what do you expect to see after having executed called`report_progress`? this should be investigated